model/device_test.go

```diff
@@ -75,3 +75,133 @@ func TestDeviceAttributesMarshal(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "[]", string(data))
 }
+
```

Comment:
general comment for the tests: do we have units that check if given attribute landed in correct scope?